src/schemas/21-526EZ/schema.js

```diff
@@ -158,6 +158,7 @@ let schema = {
     privateTreatmentCenterAddress: privateTreatmentCenterAddressDef,
     directDeposit: _.merge(definitions.bankAccount, uniqueBankFields),
     date: definitions.date,
+    dateRange: definitions.dateRange,
```

Comment:
Is this because it was still being used somewhere when it was removed?

Reply:
Yeah...turns out it's still being used in the `servicePeriodsDef` on L85